### PR TITLE
Add note about needing docker-compose v1.28+ for DFP

### LINKED ISSUE #919: [DOC]: docker-compose fails, while 'docker compose' works to build DFP production
### How would you describe the priority of this documentation request

Medium

### Please provide a link or source to the relevant docs

https://github.com/nv-morpheus/Morpheus/blob/branch-23.07/examples/digital_fingerprinting/production/README.md

### Describe the problems in the documentation

In our docs we instruct users to use `docker-compose` rather than `docker compose`.

On my system I encountered this error:
```
$ docker-compose build
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.jupyter.deploy.resources.reservations value Additional properties are not allowed ('devices' was unexpected)
services.morpheus_pipeline.deploy.resources.reservations value Additional properties are not allowed ('devices' was unexpected)
Unsupported config option for services.jupyter: 'profiles'
Unsupported config option for services.morpheus_pipeline: 'profiles'
services.jupyter.build contains unsupported option: 'target'
services.morpheus_pipeline.build contains unsupported option: 'target'
```

This could just be a version mis-match on my system, and I was using the OS installed docker not the one in the conda env:
```
$ docker --version
Docker version 20.10.5+dfsg1, build 55c4c88
$ docker-compose --version
docker-compose version 1.25.0, build unknown
```

### (Optional) Propose a correction

1) Determine if we are doing anything in `examples/digital_fingerprinting/production/docker-compose.yml` requiring a minimum version of docker compose.

2) Document which commands the user should use depending on their version of docker

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by dagardner-nv**
## Description
fixes #919

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

File: docs/source/developer_guide/guides/5_digital_fingerprinting.md [hunk 1]
```diff
@@ -232,41 +274,44 @@ Both scripts are capable of running either a training or inference pipeline for
 | `--train_users` | One of: `all`, `generic`, `individual`, `none` | Indicates whether or not to train per user or a generic model for all users. Selecting `none` runs the inference pipeline. |
 | `--skip_user` | TEXT | User IDs to skip. Mutually exclusive with `only_user` |
 | `--only_user` | TEXT | Only users specified by this option will be included. Mutually exclusive with `skip_user` |
-| `--start_time` | TEXT | The start of the time window, if undefined start_date will be `now()-duration` |
+| `--start_time` | TEXT | The start of the time window, if undefined start*date will be `now()-duration` |
```
Comment (cwharris) [inline docs/source/developer_guide/guides/5_digital_fingerprinting.md:277]: typo?

Comment (cwharris) [reply docs/source/developer_guide/guides/5_digital_fingerprinting.md:277]: probably this whole output should remain unchanged.

File: examples/digital_fingerprinting/production/README.md [hunk 2]
```diff
@@ -90,23 +113,25 @@ Both scripts are capable of running either a training or inference pipeline for
...
-| `--sample_rate_s` | INTEGER | Minimum time step, in milliseconds, between object logs.  [env var: `DFP_SAMPLE_RATE_S`; default: 0] |
-| `-f`, `--input_file` | TEXT | List of files to process. Can specify multiple arguments for multiple files. Also accepts glob (*) wildcards and schema prefixes such as `s3://`. For example, to make a local cache of an s3 bucket, use `filecache::s3://mybucket/*`. Refer to [fsspec documentation](https://filesystem-spec.readthedocs.io/en/latest/api.html?highlight=open_files#fsspec.open_files) for list of possible options. |
-| `--watch_inputs` | FLAG | Instructs the pipeline to continuously check the paths specified by `--input_file` for new files. This assumes that the at least one paths contains a wildcard. |
-| `--watch_interval` | FLOAT | Amount of time, in seconds, to wait between checks for new files. Only used if --watch_inputs is set. [default `1.0`] |
-| `--tracking_uri` | TEXT | The MLflow tracking URI to connect to the tracking backend. [default: `http://localhost:5000`] |
+| `--cache_dir` | TEXT | The location to cache data such as S3 downloads and pre-processed data [env var: `DFP_CACHE_DIR`; default: `./.cache/dfp`] |
```
Comment (cwharris) [inline examples/digital_fingerprinting/production/README.md:116]: output should probably remain unchanged.

**Review by cwharris — CHANGES_REQUESTED** [review]

**Comment by dagardner-nv** [comment]
I have no idea how/why those extra changes got in there. @cwharris thanks for catching

**Pull request closed without merging**